controllers/confidential_handler.go

```diff
@@ -28,6 +28,10 @@ const (
 	// Extended resources for TEE
 	intelTDXExtendedResource = "tdx.intel.com/keys"
 	amdSNPExtendedResource   = "sev-snp.amd.com/esids"
+
+	// INITDATA value for non-confidential peer pods, this is required in order
+	// to override the default restrictive CoCo agent policy
+	defaultInitdata = "H4sIAAAAAAAAA42UTU/DMAyG7/0VVS+7TUNwQEg7jG3AJMaqdmNH5LVeG5HGJXHGyq8npRLiQNodcrGfOH79EZAFacFlFU7DyJRwfXsTBSfURpBqTZPx1XgSBUEQ1SRF1ow1FhQ5z2g0CmrI3qHA0B3Fbx0QBDkewUoOZ3k+S+IXFEV5IG0S/LBoOLybhqwt/sXST6h97rkkgynnQnkJqpsHIdHr1wiMc1IMQqHux1JQ+YHOPmjhjJqaAWp5xizWlKHxqn5EXiNrkfURm816eXKl9SKtdYFOl/SGeRaGV4pRH8Hl00clZNlPrLF6Iq6lLe4bp+3gLfdGSVfmebxzN3xMDNYMdyS2Uq4qN1w+IEHIU3Z98z6UYEWn4Zc6LGWQ+Co0CzqatASNa7KKvSVJ0CDmiZsFqhZ46sFsNZxDitz1003hVlRe0Y6LfzbNC4hCgRyYP6dV83BOLbXVkAlV9DBsLohE9UCgLTf7dsuN+PKK39X5JbvcYcu6xAo1yP42dvDvjvRj/UuyB8EDhd+73xb/H9r2S/0GHJexXZAFAAA="
```

Comment:
Since this for non-confidential peer pods only, what about naming the constant `permissiveInitData` or something like that ?